### DISCUSSION TITLE
uYouEnhanced Discussion

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#61] [General]
This is the discussion about the tweak that made by qnblackcat and the contributors.

**Comment by arichornlover** [comment]
Hey @Dora727 I saw this https://github.com/qnblackcat/uYouPlus/issues/1202
and I’m commenting here because this fork actually has the changes right now! OLED Dark Mode is patched and this version of the tweak is using uYou v3.0.1 along with the latest version of YouTube.
and this Fork only exists for Extra changes and updates if qnblackcat’s uYouPlus doesn’t update.

**Reply by dora727** [reply]
thanks for let me know! actually someone sent it to me too lol

btw great work

**Comment by Wrestor** [comment]
Hey arichornlover, is there any specific reason why every uYou+ version after uYouPlus_18.07.5_2.3.1 will not prompt to enable notifications and subsequently disable the ability to change a channels notification setting from within the app? This is what is keeping me on that build instead of the newest one/forks of it.

**Reply by arichornlover** [reply]
I sadly can’t enable notifications or access the channel notifications on any version I go on. And since the app is the Sideloaded version that would also explain why notifications don’t work at all.
